sbat.c

```diff
@@ -316,11 +316,14 @@ check_sbat_var_attributes(UINT32 attributes)
 }
 
 bool
-preserve_sbat_uefi_variable(UINT8 *sbat, UINTN sbatsize, UINT32 attributes)
+preserve_sbat_uefi_variable(UINT8 *sbat, UINTN sbatsize, UINT32 attributes, char *sbat_var)
```

Comment:
nit: line lengths in this function are pretty long... maybe some intermediate variables would help?  Not the end of the world if it doesn't